src/main/java/uk/gov/hmcts/reform/idam/health/probe/ScheduledHealthProbeIndicator.java

```diff
@@ -59,7 +66,11 @@ protected void refresh() {
         if (probeResult || failureHandling == HealthProbeFailureHandling.MARK_AS_DOWN) {
             Status newStatus = probeResult ? Status.UP : Status.DOWN;
             if (log.isInfoEnabled() && this.status != newStatus) {
-                log.info("{}: Status changing from {} to {}", this.healthProbe.getName(), this.status, newStatus);
+                if (Status.DOWN.equals(newStatus)) {
+                    log.error("{}: Status changing from {} to {}", this.healthProbe.getName(), this.status, newStatus);
+                } else {
+                    log.info("{}: Status changing from {} to {}", this.healthProbe.getName(), this.status, newStatus);
```

Comment:
this is ok

Reply:
Some repetition though, concatenation into a final variable could be better.